handyrl/environments/tictactoe.py

```diff
@@ -55,9 +55,9 @@ def play(self, action):
 
         # check winning condition
         if self.board[x, :].sum() == 3 * self.color \
-          or self.board[:, y].sum() == 3 * self.color \
-          or (x == y and np.diag(self.board, k=0).sum() == 3 * self.color) \
-          or (x == 2 - y and np.diag(self.board[::-1, :], k=0).sum() == 3 * self.color):
+            or self.board[:, y].sum() == 3 * self.color \
+            or (x == y and np.diag(self.board, k=0).sum() == 3 * self.color) \
+                or (x == 2 - y and np.diag(self.board[::-1, :], k=0).sum() == 3 * self.color):
```

Comment:
(It's a small thing!) Why does this indent has more 4 spaces from the previous line?

Reply:
PEP8 requires this indent.

**E125 continuation line with same indent as next logical line**

https://www.flake8rules.com/rules/E125.html